resources/texts/change_log.txt

```diff
@@ -2,23 +2,35 @@
 
 * New features and improvements:
 - Introduced the MethodX and MethodXL printers with their materials and the option to write to .makerbot
-- Prime tower base settings increase the stability of large prime towers; You can tune the Size, Height, and Slope to increase your prime tower sturdiness
-- Introduced the setting for Raft Line Spacing, but for the primetower
+- Prime tower base settings increase the stability of large prime towers; You can tune the Size, Height, and Slope to increase your prime tower's sturdiness
+- Introduced the setting for Raft Line Spacing, but for the prime tower
 - The Start and End gcodes now accept variables, and math including those variables
 - Updated the Support Z Distance to allow for more different values, no longer only multiple layerheights
 
 * Bug fixes:
 - Fixed a crash for some linux users when trying to open the file menu
 
+* Bugs resolved since the Beta Release
+- Introduced Balanced and Solid profiles for 1C cores to UltiMaker Method X and Method XL
+- Improved self-support for UltiMaker MethodX and MethodXL printers so the second extruder will not be used
```

Comment:
In Makerbot jargon we don't speak about "first" and "second" extruder, but instead we talk about "model" and "support" extruder. I think it we change it the language to reflect this it makes more sense for the people for whom it is written.
```suggestion
- Improved self-support for UltiMaker MethodX and MethodXL printers so the support extruder will not be used
```